Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -4921,17 +4921,20 @@ ModelElementEnabled:
 
 ModelProcessEnabled:
   type: bool
-  status: unstable
+  status: Modelprocess_feature_status
```

Comment:
This pattern is unfortunate, but if you case sensitive search for `_feature_status`, there are five other occurrences like this one